applications/nrf5340_audio/src/bluetooth/bt_stream/le_audio.c

```diff
@@ -36,6 +36,32 @@ bool le_audio_ep_state_check(struct bt_bap_ep *ep, enum bt_bap_ep_state state)
 	return false;
 }
 
+bool le_audio_ep_qos_configured(struct bt_bap_ep *ep)
```

Comment:
ep is not changed in this function, hence:
```suggestion
bool le_audio_ep_qos_configured(struct bt_bap_ep const * const ep)
```